### PR TITLE
A couple more small tidy ups in the prim prepare code.

### PR DESCRIPTION
**Pull request by glennw**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2366)

### CODE REVIEW

File: webrender/src/frame_builder.rs [hunk 1]
```diff
@@ -144,12 +144,14 @@ pub struct FrameState<'a> {
...
 
-pub struct PictureContext {
+pub struct PictureContext<'a> {
     pub pipeline_id: PipelineId,
-    pub pic_index: SpecificPrimitiveIndex,
     pub perform_culling: bool,
     pub prim_runs: Vec<PrimitiveRun>,
     pub original_reference_frame_id: Option<ClipId>,
+    pub display_list: &'a BuiltDisplayList,
+    pub draw_text_transformed: bool,
```
Comment (kvark) [inline webrender/src/frame_builder.rs]: is it always equal to `self.original_reference_frame_id.is_some()`?

Comment (glennw) [reply webrender/src/frame_builder.rs]: For now, yes. That won't be true in the future though (once we expose to the API whether a stacking context should be drawn in screen or local space).

**Pull request merged**